sphinx/source/docs/user_guide/bokehjs.rst

```diff
@@ -280,6 +280,78 @@ and hover policy. Here is an example of a ``bar`` chart and the plot it generate
 
     plt.show(plt.gridplot([[p1, p2], [p3, p4]], {plot_width:350, plot_height:350}));
 
+.. _userguide_bokehjs_differences:
+
+Usage : Comparison to Python
+----------------------------
+
+While the BokehJS models generally match the Python equivalents, there are some cases
+where usage can differ significantly between BokehJS and the Python equivalent. Some
+instances we have identified are listed below.
+
+.. _userguide_bokehjs_differences_titles:
+
+Updating figure titles
+~~~~~~~~~~~~~~~~~~~~~~
+
+Calling ``var plot = new Bokeh.Plotting.figure({title:"some title"})`` replaces plot.title (a ``Bokeh.Title()``
+object by default) with the provided string. This makes it so the title cannot be dynamically updated later.
+In Python, the equivalent approach casts the string to the underlying Title() object.
+
+To preserve the ability to update the title later, use the following sequence instead:
+
+``var plot = new Bokeh.Plotting.figure()``
```

Comment:
For code blocks, please use the `code-block` directive:

  .. code-block:: javascript
    var plot = new Bokeh.Plotting.figure()
    plot.title.text = "some title"